WcaOnRails/app/models/competition.rb

```diff
@@ -54,6 +55,13 @@ def must_have_at_least_one_delegate
     end
   end
 
+  validate :delegates_must_be_delegates, on: [:update]
```

Comment:
Because when creating a competition we only pass the id...it would fail the validation.
Also, the cloning happens before validation, so I think it is fine.